NT-Line-team/Behave_test
# Tamho30

## Exercise
Excercise #30 
* [Exercise-xx](https://github.com/NT-Line-team/Behave_test/issues/<issue_number>)

## Overview
tamhom-issue30: Implement ticket for the exercise30 "Print out an interesting square with O, + and -"

## Verification Steps
This exercise is for practicing for loop and if statements.

Write the program that allow user to input N, then print the square as below:

Please input N:10

O  O  O  O  O  O  O  O  O  O  
O  O  -    -   -   -    -    -   -  O 

O  +  O  -    -   -    -    -   - O 

O  +  +  O  -    -    -    -   - O 

O  +  +  +  O   -    -    -   - O 

O  +  +  +  +  O  -    -   -   O 

O  +  +  +  +  +  O  -    -  O 

O  +  +  +  +  +  +  O  -   O 

O  +  +  +  +  +  +  +  O  O 

O  O  O  O  O  O  O  O  O  O 

1. `git checkout pr/###`
2. Check the functionality

### Docs
- [ ] Are changes required for docs?
    - [ ] `README.md` was updated?
    - [ ] Additional documentation required for `docs/`?
